src/styles/blocks/card.scss

```diff
@@ -0,0 +1,128 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: $white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transform: scale(120%);
+    transition: all $duration;
+
+    & &__product-name {
```

Comment:
The selector `& &__product-name` seems incorrect. It should be `&__product-name` to target the child element correctly within the hover state.